libs/gltfio/include/gltfio/FilamentAsset.h

```diff
@@ -77,6 +77,10 @@ class FilamentAsset {
     size_t getTextureBindingCount() const noexcept;
     const TextureBinding* getTextureBindings() const noexcept;
 
+    /** Gets resource URLs for all externally-referenced buffers. */
+    size_t getResourceUrlCount() const noexcept;
```

Comment:
Rename to `getResourceUriCount` and `getResourceUris` for consistency? I think this is what the official spec refers to them as. (Also update comment here and other locations in this PR)

Reply:
Good point, added a commit for this renaming.